packages/amplify-graphql-searchable-transformer/src/__tests__/__snapshots__/amplify-graphql-searchable-transformer.tests.ts.snap

```diff
@@ -101,14 +101,49 @@ input SearchableEmployeeSortInput {
   direction: SearchableSortDirection
 }
 
+input SearchableEmployeeAggregationInput {
+  name: String!
+  type: SearchableAggregateType!
+  field: String!
+}
+
+enum SearchableAggregateType {
+  terms
```

Comment:
Do we generate other enums with lowercase entries? I think enums are generally all capital letters, so we should do that here unless this was done for consistency. The RFC uses all caps entries.

Reply:
Yes, we use lower case in other places too. Example: SearchableSortDirection.
Also, OpenSearch expects this in lower case.